harvester_e2e_tests/integrations/test_upgrade.py

```diff
@@ -547,9 +554,44 @@ def test_degraded_volume(
 @pytest.mark.any_nodes
 class TestAnyNodesUpgrade:
     @pytest.mark.dependency(name="preq_setup_logging")
-    def test_preq_setup_logging(self, api_client):
-        # TODO: enable addon if > v1.2.0
-        return
+    def test_preq_logging_pods(self, api_client, wait_timeout, sleep_timeout):
+        # logging is an addon instead of built-in since v1.2.0
+        if api_client.cluster_version.release >= (1, 2, 0):
+            addon = "cattle-logging-system/rancher-logging"
+
+            code, data = api_client.addons.get(addon)
+            assert 200 == code, (code, data)
+            assert not data.get('spec', {}).get('enabled', True), (code, data)
```

Comment:
IMO, we can just skip the set up when the `addon` had been enabled, it would let us to reuse existing cluster to do the upgrade testing.

another reason is, we are not testing the setting, we setting up the environment to hit the prerequisite.

Reply:
Enhanced in 6a19ea8 with `logging_addon` fixture.